api-reference/10 UI Widgets/dxPopover/1 Configuration/position.md

```diff
@@ -16,4 +16,4 @@ Besides the position configuration object, the option can take on the following
 - "left" - places the popover to the left of the target element
 - "right" - places the popover to the right of the target element
 
-[note]To specify the element against which the widget will be positioned, use the [target](/api-reference/10%20UI%20Widgets/dxPopover/1%20Configuration/target.md '{basewidgetpath}/Configuration/#target') option instead of the [of](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') field of the **position** object. If both **target** and **of** are specified, **target** will be used.
\ No newline at end of file
+You can use the [position](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#position).[of](/api-reference/50%20Common/Object%20Structures/positionConfig/of.md '/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of') option instead of the [target](/Documentation/ApiReference/UI_Widgets/dxPopover/Configuration/#target) option to specify the element against which the widget will be positioned. If both **target** and **position**.**of** are specified, the latter will take precedence. However, if the **of** field of the **position** object is not specified, the **target** option will be used.
```

Comment:
Should we use 'instead of' here? Target option determines not only the default position of the popover, but also the element, the interaction with which opens the popover

Reply:
Ok, I see the point, but here we talk only about the popover's position. Anyway, I'll try to rewrite this paragraph to avoid ambiguity.

Reply:
Thanks